core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -263,13 +263,18 @@ coreos:
       command: start
       content: |
         [Unit]
-        Requires={{.Etcd.SystemdUnitName}}
+        Wants={{.Etcd.SystemdUnitName}}
         After={{.Etcd.SystemdUnitName}}
 
         [Service]
-        Type=oneshot
+        Type=simple
+        Restart=on-failure
+        RestartSec=10
+
         EnvironmentFile={{.EtcdNodeEnvFileName}}
-        ExecStart=/opt/bin/cfn-signal
+        ExecStartPre=/usr/bin/systemctl is-active {{.Etcd.SystemdUnitName}}
+        ExecStartPre=/usr/bin/rkt fetch --pull-policy new {{.AWSCliImage.Options}}{{.AWSCliImage.RktRepo}}
```

Comment:
`--pull-policy` doesn't seem to exist in rkt on the latest Container Linux stable.

```
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal systemd[1]: cfn-signal.service: Service hold-off time over, scheduling restart.
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal systemd[1]: Stopped cfn-signal.service.
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal systemd[1]: Starting cfn-signal.service...
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal systemctl[5552]: active
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]: Error: unknown flag: --pull-policy
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]: NAME:
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:         fetch - Fetch image(s) and store them in the local store
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]: USAGE:
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:         rkt fetch IMAGE_URL...
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]: DESCRIPTION:
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:         Locates and downloads remote ACIs and their attached signatures.
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:         If the ACI is available in the local store, the image will not be fetched
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:         again.
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]: OPTIONS:
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:       --full[=false]                print the full image hash after fetching
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:       --no-store[=false]        fetch images ignoring the local store
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:       --signature=                local signature file to use in validating the preceding image
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:       --store-only[=false]        use only available images in the store (do not discover or download from remote URLs)
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]: GLOBAL OPTIONS:
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:       --debug[=false]                        print out more debug information to stderr
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:       --dir=/var/lib/rkt                rkt data directory
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:       --insecure-options=none                comma-separated list of security features to disable. Allowed values: "none", "image"
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:       --local-config=/etc/rkt                local configuration directory
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:       --system-config=/usr/lib/rkt        system configuration directory
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:       --trust-keys-from-https[=false]        automatically trust gpg keys fetched from https
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal rkt[5563]:       --user-config=                        user configuration directory
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal systemd[1]: cfn-signal.service: Control process exited, code=exited status=254
Apr 20 10:17:11 ip-10-0-0-92.ap-northeast-1.compute.internal systemd[1]: Failed to start cfn-signal.service.
```

```
core@ip-10-0-0-92 ~ $ rkt version
rkt Version: 1.21.0
appc Version: 0.8.9
Go Version: go1.7.3
Go OS/Arch: linux/amd64
Features: -TPM +SDJOURNA
```